cluster-service/deploy/helm/values.yaml

```diff
@@ -59,7 +59,7 @@ gatewayUrl: "http://127.0.0.1:9090"
 clientScopes: "openid"
 
 # Environment associated with this instance.
-environment: "aro-hcp-dev"
+environment: ""
```

Comment:
Does this mean that by default it is the "" empty string? Is this intentional?

CS will fail to start if that's not set. I understand that we are setting this variable externally from the Makefile. If someone were to run the helm chart directly I understand it would fail.

This might be intentional and we might just want to be it that way. I am raising it just to be sure we are intentional/aware of it.

Reply:
this is a pattern. good that CS will fail. it means that we will spot an error if we fail to pass something valid to helm